File: app/src/main/java/com/udacity/popularmovies/model/MovieTrailer.java
package com.udacity.popularmovies.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class MovieTrailer {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("results")
    @Expose
    private ArrayList<TrailerDetails> results = null;

    private ArrayList<String> url= null;


    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public ArrayList<TrailerDetails> getResults() {
        return results;
    }

    public void setResults(ArrayList<TrailerDetails> results) {
        this.results = results;
}

    public ArrayList<String> getUrl() {
        return url;
    }

    public void setUrl(ArrayList<String> url) {
        this.url = url;
    }
}

File: app/src/main/java/com/udacity/popularmovies/model/MovieTrailer.kt
package com.udacity.popularmovies.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

class MovieTrailer {

    @SerializedName("id")
    @Expose
    var id: Int? = null

    @SerializedName("results")
    @Expose
    lateinit var results: ArrayList<TrailerDetails>
    var url: ArrayList<String>? = null

}